index.html

```diff
@@ -152,18 +179,21 @@ <h2 class="outro-head">About Myself</h2>
                 </ul>
             </div>
         </section>
-        <section class="contact">
+        <section id="contact">
             <div class="form">
-                <img src="./images/contact form shapes-mobile.svg" alt="" id="form-background">
-                <h2 class="form-head">Contact me</h2>
-                <p>If you have an application you are interested in developing, a feature that you need built or a
-                    project that needs coding. I’d love to help with it!</p>
-                <form action="https://formspree.io/f/mayzgpbp" method="POST">
-                    <input type="text" name="fullname" maxlength="30" placeholder="Lorem Ipsum" required>
-                    <input type="email" name="email" placeholder="loremipsum@gmail.com" required>
-                    <textarea name="message" id="message" placeholder="Write your message here..." cols="30"
-                        maxlength="500" rows="7" required></textarea>
-                    <button type="submit" id="submit" class="see-project">Get in touch</button>
+                <img src="./images/contact form shapes-mobile.svg" alt="background" id="form-background">
+                <img src="./images/contact form background shapes.svg" alt="background" id="form-desktop">
+                <div class="formtemp">
+                    <h2 class="form-head">Contact me</h2>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a
+                        project that needs coding. I’d love to help with it!</p>
+                    <form action="https://formspree.io/f/mayzgpbp" method="POST">
+                        <input type="text" name="fullname" maxlength="30" placeholder="Lorem Ipsum" required>
+                        <input type="email" name="email" placeholder="loremipsum@gmail.com" required>
+                        <textarea name="message" id="message" placeholder="Write your message here..." cols="30"
+                            maxlength="500" rows="6" required></textarea>
+                        <button type="submit" id="submit" class="see-project">Get in touch</button>
+                </div>
                 </form>
             </div>
```

Comment:
[OPTIONAL]
- [ ] Please move the `form` closing tag one line up, as your div `formtemp` should wrap the whole `form` inside of it.

Reply:
Thanks. This explains why I have hard time doing css on it